src/test/java/uk/gov/hmcts/cmc/claimstore/services/document/DocumentManagementBackedDocumentsServiceTest.java

```diff
@@ -391,6 +394,34 @@ public void shouldGetOrderDocumentsForOrderDirections() {
         verify(documentManagementService, once()).downloadDocument(any(), any());
     }
 
+    @Test(expected = DocumentDownloadForbiddenException.class)
+    public void generateDocumentShouldThrowExceptionWhenExceptionIsThrown() {
+        final ClaimDocumentCollection claimDocumentCollection = new ClaimDocumentCollection();
+        Claim claim = Claim.builder()
+            .externalId("externalID")
+            .submitterId(CLAIMANT.getUserDetails().getId())
+            .claimDocumentCollection(claimDocumentCollection)
+            .build();
+
+        when(documentManagementBackendDocumentsService.generateDocument(claim.getExternalId(), SEALED_CLAIM, AUTHORISATION))
+            .thenThrow(DocumentManagementException.class);
+
+        boolean exceptionThrown = false;
+
+        try {
```

Comment:
instead of this try catch block here, please use `@Rule` i.e. 
```@Rule
public ExpectedException exceptionRule = ExpectedException.none();

@Test
public void whenExceptionThrown_thenRuleIsApplied() {
    exceptionRule.expect(DocumentManagementException.class);
    exceptionRule.expectMessage("exception message here");
    ..... more code here ... 
} ```

or if we have junit5 `assertThrows` can be used instead